app/scripts/modules/core/src/managed/ArtifactDetail.less

```diff
@@ -0,0 +1,15 @@
+.ArtifactDetail {
+  background-color: var(--color-white);
+  padding: 16px 0 64px 16px;
+
+  .detail-section-left {
+    flex: 0 0 280px;
+    font-style: italic;
+    font-size: 14px;
+    color: var(--color-charcoal);
+  }
+
+  .detail-section-right {
+    flex: 1 1 auto;
+  }
+}
```

Comment:
because i've already been working with stuff in `core/presentation` that uses some of the new colors/styles, it was quick and straightforward for me to move the CSS module code into `.less` files, change the class names, and replace things like flex layout rules + text weights + colors with standard stuff. I don't think that's 100% necessary right now but I did it.